### PR TITLE
Feat/stats page

### PR DESCRIPTION
**Pull request by belljustin**
Refs #57

### CODE REVIEW

File: diamm/serializers/website/stats.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import serpy
+
+from diamm.serializers.serializers import ContextDictSerializer
+
+class StatsSerializer(ContextDictSerializer):
+    source = serpy.IntField(label='sources')
```
Comment (ahankinson) [inline diamm/serializers/website/stats.py]: This is redundant. The `label` property is only useful if its value is different from the field name. Think "type =" but you want the serialized field to be "@type" (which is an illegal field name in Python).

File: diamm/templates/stats.jinja2 [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{% extends "base.jinja2" %}
+
+{% block body %}
+<div class="row">
+    {% for key, value in content.items() %}
```
Comment (ahankinson) [inline diamm/templates/stats.jinja2]: You will want to make sure you include the grid system here.

File: diamm/serializers/website/stats.py (hunk 1 above)
Comment (belljustin) [inline diamm/serializers/website/stats.py]: I was using the label field as a simple way to make sure the titles were pluralized. Should I use a filter in the template instead? That approach would be less fun when we encounter titles that don't just append an 's' e.g. person -> people

Comment (ahankinson) [inline diamm/serializers/website/stats.py]: What's wrong with `sources = serpy.IntField()`?

Comment (belljustin) [inline diamm/serializers/website/stats.py]: I was under the impression that the variable name was used to lookup the value in the dict that's passed to the serializer. Wouldn't `sources = serpy.IntField()` result in a KeyError

Comment (belljustin) [inline diamm/serializers/website/stats.py]: Or are you suggesting that I rename the keys in the view before I pass them to the serializer?

Comment (ahankinson) [inline diamm/serializers/website/stats.py]: `sources = serpy.IntField(attr="source")`?

Comment (ahankinson) [inline diamm/serializers/website/stats.py]: This pattern is used all over the place in the other serializers: https://github.com/diamm/diamm/blob/master/diamm/serializers/search/source.py#L24

**Pull request merged**